modules/dreamview/backend/teleop/teleop.cc

```diff
@@ -84,6 +111,7 @@ void TeleopService::RegisterMessageHandlers() {
         {
           boost::unique_lock<boost::shared_mutex> writer_lock(mutex_);
           teleop_status_["video"] = !teleop_status_["video"];
+          AINFO << "ToggleVideo" << std::endl;
```

Comment:
std::endl is not needed for logging output

Reply:
OK, I removed all of them.